source/includes/extracts-atlas-search-facts.yaml

```diff
@@ -0,0 +1,24 @@
+ref: fact-text-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. A legacy text search capability is available for users 
+  self-managing MongoDB deployments. 
+
+---
+ref: fact-regex-search-legacy-atlas
+content: |
+
+  MongoDB offers a :atlas:`premium full-text search solution, MongoDB 
+  Atlas Search </atlas-search/>`, for data hosted on :atlas:`MongoDB 
+  Atlas </>`. If you frequently run case-insensitive regex queries 
+  (utilizing the ``i`` option), we recommend Atlas Search queries that 
+  use the :pipeline:`$search` aggregation pipeline stage. :atlas:`Atlas 
+  Search queries </reference/atlas-search/searching/>` significiantly 
+  improve the performance of text queries and offer more options for 
+  customizing query parameters. To learn more, see :atlas:`MongoDB 
+  Atlas Search </atlas-search/>`. For self-managed deployments, you 
```

Comment:
Consider separating this out with bullets, one for Atlas-managed deployments and one for self-managed deployments.

```
- For data hosed on MongoDB Atlas, MongoDB offers...

- For self-managed deployments, you can...
```